osu.Game/Online/Metadata/IMetadataClient.cs

```diff
@@ -9,7 +9,7 @@ namespace osu.Game.Online.Metadata
     /// <summary>
     /// Interface for metadata-related remote procedure calls to be executed on the client side.
     /// </summary>
-    public interface IMetadataClient : IStatefulUserHubClient
+    public partial interface IMetadataClient : IStatefulUserHubClient
```

Comment:
In terms of mechanics I'd prefer a single `MetadataClient` implementing multiple interfaces rather than one partial `IMetadataClient` interface. I understand the reasoning for why the *implementation* isn't split off (signalr design paradigms forcing hubs to live under different urls / requiring separate websockets, and we already have too many to manage), but the abstraction layer (as in the interfaces, etc.) I'd prefer to be kept as separate as feasible.